styles/global.css

```diff
@@ -23,6 +23,30 @@
   font-display: swap;
 }
 
+@font-face {
+  font-family: 'Segoe';
+  src: url('/assets/fonts/SegoeUI.ttf');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Roboto';
+  src: url('/assets/fonts/Roboto-Regular.ttf');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'SF-Pro';
+  src: url('/assets/fonts/SFProDisplay-Light.ttf');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
```

Comment:
Are these still needed even with the component coming from NPM?

Reply:
Nevermind, I see they're used above. Thanks.

Reply:
Yeah, the default font family is Arial but I use CSS variables for customizing the fonts for each platform.
Read the docs hehe :P